test/format-test.js

```diff
@@ -76,7 +76,7 @@ vows.describe('syslog messages').addBatch({
             transport = new winston.transports.Syslog({
               port: PORT,
               type: '5424',
-              appId: 'hello'
+              appName: 'hello'
```

Comment:
Maybe adding a copy of this test that sets `app_name` too and see if it also works